pipeline/schemas/file.py

```diff
@@ -13,6 +13,7 @@ class FileFormat(str, enum.Enum):
 
 class FileBase(BaseModel):
     name: str
+    # TODO(RG): hex is currently the default but this will soon change
```

Comment:
think this line was left over from a previous commit of mine (you can probably remove it)